enable-vcp-image/enable-vcp-scripts/daemonset_pod.sh

```diff
@@ -70,21 +98,27 @@ if [ $? -eq 0 ]; then
     if [ $? -eq 0 ]; then
         echo "[INFO] Existing vsphere.conf file is copied to" /host/tmp/$POD_NAME/vsphere.conf
     else
-        echo "[ERROR] Failed to back up vsphere.conf file at " /host/tmp/$POD_NAME/vsphere.conf
+        ERROR_MSG="Failed to back up vsphere.conf file at " /host/tmp/${POD_NAME}/vsphere.conf
+        echo "[ERROR] ${ERROR_MSG}"
+        update_VcpConigStatus "$POD_NAME" "$PHASE" "FAILED" "$ERROR_MSG"
         exit $ERROR_FAIL_TO_BACKUP_FILE
     fi
 fi
 
 # locate and back up manifest files and kubelet service configuration file.
 file=/host/$k8s_secret_kubernetes_api_server_manifest
 backupdir=/host/tmp/$POD_NAME
-locate_validate_and_backup_files $file $backupdir
+locate_validate_and_backup_files $file $backupdir $POD_NAME
 
 file=/host/$k8s_secret_kubernetes_controller_manager_manifest
-locate_validate_and_backup_files $file $backupdir
+locate_validate_and_backup_files $file $backupdir $POD_NAME
 
 file=/host/$k8s_secret_kubernetes_kubelet_service_configuration_file
-locate_validate_and_backup_files $file $backupdir
+locate_validate_and_backup_files $file $backupdir $POD_NAME
```

Comment:
Inside locate_validate_and_backup_files, update_vcpStatus is called only when error happens...
So I am not sure if they can be moved to the main flow here.
But I do see that the following pattern happens everywhere:
```
                ERROR_MSG="Failed to Validate JSON for file:" $CONFIG_FILE
                echo "[ERROR] ${ERROR_MSG}"
                update_VcpConigStatus "$POD_NAME" "$PHASE" "FAILED" "$ERROR_MSG"
```
Is it possible to make another utility error print function to include these three lines?
Or, maybe move the echo into update_VcpConigStatus, and depending on the status is "FAILED" or "RUNNING", echo message with different tag, [ERROR] or [INFO].